config/webpack.config.js

```diff
@@ -198,7 +203,7 @@ const config = {
 			),
 		// Lint CSS.
 		new StyleLintPlugin({
-			context: path.resolve(process.cwd(), './assets/css'),
+			context: path.resolve(process.cwd(), configPaths.srcDir),
```

Comment:
Is this path correct?

Reply:
I believe so as we may have CSS in the src dir folder.